build/jobs/run-tests.yml

```diff
@@ -0,0 +1,133 @@
+parameters:
+- name: version
+  type: string
+- name: keyVaultName
+  type: string
+- name: runIntegration
+  type: boolean
+  default: true
+- name: runE2E
+  type: boolean
+  default: true
+
+jobs:
+- job: "integrationTests"
+  condition: eq('${{ parameters.runIntegration }}', true)
+  pool:
+    vmImage: $(WindowsVmImage)
+  steps:
+  - task: DownloadBuildArtifacts@0
+    inputs:
+      buildType: 'current'
+      downloadType: 'single'
+      downloadPath: '$(System.ArtifactsDirectory)'
+      artifactName: 'IntegrationTests'
+
+  - task: UseDotNet@2
+    inputs:
+      useGlobalJson: true
+
+  - task: AzureKeyVault@1
+    displayName: 'Azure Key Vault: ${{ parameters.keyVaultName }}'
+    inputs:
+      azureSubscription: 'Microsoft Health Open Source Subscription'
```

Comment:
$(ConnectedServiceName)

Reply:
I'll fix this in the yaml PR. It'll show up here when I merge from master.